install.sh

```diff
@@ -1161,15 +1179,27 @@ set_githooks_directory() {
         git config --global githooks.useCoreHooksPath yes
         git config --global githooks.pathForUseCoreHooksPath "$1"
         git config --global core.hooksPath "$1"
+
+        CURRENT_CORE_TEMPLATE_DIR=$(git config --global init.templateDir)
+        # shellcheck disable=SC2012
+        if [ "$(ls -1 "$CURRENT_CORE_TEMPLATE_DIR/hooks" 2>/dev/null | wc -l)" != "0" ]; then
```

Comment:
This will almost always be true for regular git installations, since that comes with a bunch of sample files in there.
I believe git will use `core.hooksPath` first if it's set, so masking the hooks in the template dir is probably not that big of a deal, than on the opposite side.
I'd remove this error as I don't think it would help people much to be honest.
What do you think @gabyx ?

Reply:
Ahh you are right. That error will then be always visible when somebody is using core.hooksPath with a normal installation. Not good hm... I will check with the git manual. or test it if `git config --global init.templateDir` always resolvs to the standard installation default... Good point

Reply:
OK, as the dokumentation says and what I see here on the terminal. `core.initTemplateDir` [--local, --global, --system] is simply set by the user, it does not resort to the default location of a git install. So this error would be helpful I think.
We should also test for the "$GIT_TEMPLATE_DIR" as this could be set as well...

https://git-scm.com/docs/git-init

Reply:
Oh, did I confuse this with the default path at `/usr/share/git-core/templates` ?
> the init.templateDir configuration variable; or
> the default template directory: /usr/share/git-core/templates.

If this is the case on popular environments, then I'm fine with it.
- OS X (not sure how we can test this)
- Windows / Git Bash - can you test this?
- Debian / Ubuntu / Arch maybe

Reply:
Yes sorry variable naming is bad, changed it in my next commit.

Reply:
This warning is exactly what it says :-). I am not so eager to write a test just for this warning. ;-) Hm... or what did you thing should be tested?

Reply:
Whether `init.templateDir` is set on a default installation. If you're confident it's not then OK, for some reason I thought it is set on Debian or Ubuntu. Let me see if I can check.

Reply:
core.hooksPath is a complete override for anything :-) which gets installed with any kind of template dir used for git init 
https://git-scm.com/docs/git-init

Reply:
OK, I was wrong, it uses that default path on Ubuntu and Debian it seems. I guess it will be the same on Git Bash, no idea about OS X and homebrew, but let's assume it'll work it out without setting `init.templateDir`?

Reply:
On Linux (Ubuntu here) its available, I mean we only cover the specs of git, we also only try to use it if its available `mark_directory_as_target "/usr/share/git-core/templates/hooks" && return 0` 
and I dont know what how this review thread is related to the code you tagged?

Reply:
I initially thought some Git installations come with a global `init.templateDir` set, but it turns out I was wrong, that's where this thread came from. It sounds like it's all good, nothing to change here. 👍